project/src/com/company/services/BankService.java

```diff
@@ -0,0 +1,130 @@
+package com.company.services;
+
+import com.company.entities.*;
+
+import java.util.Collections;
+import java.util.Objects;
+import java.util.Vector;
+
+public class BankService {
+    static BankService instance = null;
+    static String status = "Not created";
+    Vector<Account> accounts;
+
+    //get instance
+    public static BankService getInstance()
+    {
+        if (instance == null) {
+            status = "Created";
+            instance = new BankService();
+        }
+        return instance;
+    }
+
+    //constructor
+    public BankService() {
+        this.accounts = new Vector<Account>();
+    }
+
+    //Create
+    public void addAccount(boolean is_premium, String first_name, String last_name, String email){
+        Account account = null;
```

Comment:
no need to instantiate it with null, just declare it

Reply:
Fixed.